src/partials/layout.html

```diff
@@ -221,5 +221,54 @@
       });
     </script>
     <script async src="https://cdn.announcekit.app/widget-v2.js"></script>
+    <script>
+      filterSelection("all")
```

Comment:
Since we want to show all elements by default, instead of making them hidden and calling `filterSelection("all")` to show them, it would be better to have them shown and update `filterSelection` to `"hide"` elements that shouldn't be shown. This way we don't need for the JavaScript to be loaded and executed to show the elements :)